### ISSUE TITLE
Abount bdd100k

### ISSUE THREAD

**Issue by BruceYu-Bit** [MCG-NJU/MeMOTR#8]
Congradulations about the achievement. But i wonder when will release the bdd100k model and train methods?

**Comment by HELLORPG** [comment]
Hi, thank you for your interest in our work.

As the BDD100 evaluation server has some [issues](https://github.com/bdd100k/bdd100k/issues/322), we can't double-check our model and training scripts on BDD100K. This hinders our open-source progress because the code of the BDD100K part needs to be merged into this repo.

I'll release the code and scripts of the BDD100K dataset this weekend (as soon as possible). However, I can't rigorously check the model's outputs. If you have any problem, feel free to ask me.

**Comment by HELLORPG** [comment]
Hi, I have now uploaded all the code, configuration, and checkpoint file for BDD100K.
Please note that I'm currently unable to double-check the absolute correctness of all these. Although the calculation code for `TETA` is provided in [this repo](https://github.com/SysCV/tet/tree/main/teta), I'm still having some trouble:

1. There are some issues with running this code directly, even using the output files we already submitted to the [evaluation server](https://eval.ai/web/challenges/challenge-page/1989/leaderboard/4695).
2. After I fixed the code, the evaluation results were a little different (about 0.4 TETA) from the results from the [eval.ai](https://eval.ai/web/challenges/challenge-page/1989/leaderboard/4695).

Moreover, our experiment on BDD100K was rushed (to be precise, only one chance because it costs about one week, which is SO LONG), so I did not tune the training strategy very well (primarily refer to [MOTR](https://github.com/megvii-research/MOTR)). Therefore, there may be some known instability factors. Most likely, it is about the difference in training and inference image size. The max image size during training is 800x1333 as we want to save the training memory and time. However, the max image size during inference is 800x1536. In our experiments, using 800x1536 for inference could achieve better tracking performance, but it may also cause convergence instability because there is a gap in position encoding between training and inference. Based on experience, **I think the model can achieve better tracking performance if the max image size `1536` (like we did for DanceTrack/SportsMOT) is also adopted during the training stage.** But I do not have enough spare GPUs for such a long training now.

**In short, if you encounter any problems in the BDD100 experiments, please feel free to contact me and discuss them together.**

**Comment by HELLORPG** [comment]
As I haven't received your reply for a long time, I am closing this issue temporarily.
Feel free to re-open this issue if you need~
